### PR TITLE
Document "ignored-exceptions-for-type" configuration option.

### LINKED ISSUE #3675: Document IgnoredExceptionsForType for Java/Android
## Issue

Related to https://github.com/getsentry/sentry-java/pull/1352/files
Java/Android supports IgnoredExceptionsForType but its not documented under https://docs.sentry.io/platforms/java/configuration/options/ and Android docs

## Description

option to be documented

### Suggested Solution

Add to default options

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
Fixes #3675

### CODE REVIEW

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -209,6 +209,12 @@ This option can be overridden using <PlatformIdentifier name="in-app-include" />
 
 </ConfigKey>
 
+<ConfigKey name="ignored-exceptions-for-type" supported={["java", "android"]}>
```
Comment (marandaneto) [inline src/platforms/common/configuration/options.mdx:212]: iirc C# already has this feature, isn't documented for C# yet, or is it another name?

**Review by marandaneto — APPROVED** [review]
LGTM

File: src/platforms/common/configuration/options.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/platforms/common/configuration/options.mdx:212]: I don't see `ExceptionFilters` in the current content, can we add that as well?

Comment (bruno-garcia) [reply src/platforms/common/configuration/options.mdx:212]: @PeloWriter recently refactored the .NET Docs, this is now here: https://docs.sentry.io/platforms/dotnet/usage/#ignoring-errors

Comment (bruno-garcia) [reply src/platforms/common/configuration/options.mdx:212]: Though in C# it's only through the API not via String names of types

**Review by bruno-garcia — APPROVED** [review]
LGTM

File: src/platforms/common/configuration/options.mdx (hunk 1 above)
Comment (marandaneto) [reply src/platforms/common/configuration/options.mdx:212]: we should have aligned the naming, its called `addIgnoredExceptionForType` on Java, should we rename it in the next major?

**Pull request merged**